packages/app/src/components/Button.tsx

```diff
@@ -2,24 +2,25 @@ import clx from "classnames";
 import React, { ReactNode } from "react";
 
 interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
-  variant?: "primary" | "secondary";
+  variant?: "green" | "red" | "unactive";
   children?: ReactNode;
   className?: string;
 }
 
 const Button: React.FC<ButtonProps> = ({
   children,
   className,
-  variant = "primary",
+  variant = "green",
   ...rest
 }) => {
   return (
     <button
       className={clx(
         "px-8 py-4 rounded-lg text-white text-xl",
         {
-          "bg-green": variant === "primary",
-          "bg-outset": variant === "secondary",
+          "bg-green": variant === "green",
+          "bg-red-400": variant === "red",
```

Comment:
Please add a color to tailwind.config.js named "danger" and use "bg-danger" here

Reply:
Done.